_apps/task-lint.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Task-Lint
+description:  Status Check that checks comments in your pull requests and will fail if any of them have tasks outstanding.
+slug: task-lint
+screenshots:
+- https://avatars3.githubusercontent.com/in/15182?s=88&amp;v=4
```

Comment:
For this screenshot, I'd prefer if you'd use the other one from down below. This is where the screenshots appear for each app, see: https://probot.github.io/apps/stale/ for an example.